app/javascript/components/subject/MoveToHousehold.js

```diff
@@ -1,69 +1,221 @@
+          { field: 'state_local_id', label: 'State/Local ID', isSortable: true, tooltip: null },
+          { field: 'jurisdiction', label: 'Jurisdiction', isSortable: true, tooltip: null },
+          { field: 'dob', label: 'Date of Birth', isSortable: true, tooltip: null, filter: this.formatDate },
+          { field: 'select', label: '', isSortable: false, tooltip: null, filter: this.createSelectButton, className: 'text-center', onClick: this.submit },
+        ],
+        rowData: [],
+        totalRows: 0,
+        selectedRows: [],
+        selectAll: false,
+      },
+      query: {
+        page: 0,
+        search: '',
+        entries: 5,
+        workflow: 'all',
+        tab: 'all',
+        scope: 'all',
+        tz_offset: new Date().getTimezoneOffset(),
+        // This query should always filter out records that are not self-reporters
+        filter: [
+          {
+            dateOption: null,
+            filterOption: {
+              description: 'Monitorees that are a Head of Household or self-reporter',
+              name: 'hoh',
+              title: 'Daily Reporters (Boolean)',
+              type: 'boolean',
+            },
+            value: true,
+          },
+        ],
+      },
+      entryOptions: [5, 10],
+      isLoading: false,
       updateDisabled: true,
       showModal: false,
-      loading: false,
-      dependents: [],
+      cancelToken: axios.CancelToken.source(),
     };
-    this.toggleModal = this.toggleModal.bind(this);
-    this.handleChange = this.handleChange.bind(this);
-    this.submit = this.submit.bind(this);
-    this.getResponders = this.getResponders.bind(this);
   }
 
-  toggleModal() {
-    let current = this.state.showModal;
-    this.setState({
-      updateDisabled: true,
-      showModal: !current,
-      loading: !current,
-    });
-    if (!current) {
-      this.getResponders();
-    }
-  }
+  /**
+   * Creates a link and renders HoH badge for monitoree name in table.
+   * @param {Object} data - provided by CustomTable about each cell in the column this filter is called in.
+   */
+  renderPatientName = data => {
+    const name = data.value;
+    const rowData = data.rowData;
 
-  handleChange(event) {
-    let updateDisabled = true;
-    if (event.target.id == 'hoh_selection') {
-      updateDisabled = event.target.value === -1;
+    if (rowData.is_hoh) {
+      return (
+        <div>
+          <BadgeHOH patientId={rowData.id.toString()} customClass={'badge-hoh ml-1'} location={'right'} />
+          <a href={`/patients/${rowData.id}`}>{name}</a>
+        </div>
+      );
     }
-    this.setState({ [event.target.id]: event.target.value, updateDisabled: updateDisabled });
+    return <a href={`/patients/${rowData.id}`}>{name}</a>;
+  };
+
+  /**
+   * Creates a "Select" button for each row of the table.
+   * @param {string} patientId
+   */
+  createSelectButton(_, patientId) {
+    return (
+      <Button id={patientId} variant="primary" size="lg">
```

Comment:
thoughts on making this button `md` or `sm` for the rows to be shorter?

Reply:
Good thought. `md` looks fine to me, don't want to go much smaller. 
![Screen Shot 2021-01-21 at 2 17 46 PM](https://user-images.githubusercontent.com/11698457/105400878-7dc25380-5bf3-11eb-9207-71281e01a80d.png)

Updated here: fa73fb4